src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -288,11 +289,16 @@ export default class StyleConfigurator {
         }
       } else if (bpmnElement instanceof ShapeBpmnSubProcess) {
         styleValues.set(StyleIdentifier.BPMN_STYLE_SUB_PROCESS_KIND, bpmnElement.subProcessKind);
-        styleValues.set(StyleIdentifier.BPMN_STYLE_IS_EXPANDED, String(bpmnCell.isExpanded));
+        if (!bpmnCell.isExpanded) {
+          markers.push('expand');
```

Comment:
:information_source: I initially planned to create a constant for that, but finally I am not sure this is valuable. What do you think about it @csouchet ?

Reply:
I tought that we remove the **isExpanded** field, and put it in the markers array of ShapeBpmnElement as a new Enum.

Reply:
as discussed, we will manage this in a dedicated PR
The model objects will be updated to store several markers. The isExpanded field will be removed and replaced by an expand marker.